schedule/sles4sap/ensa/ensa_nodes.yaml

```diff
@@ -49,6 +49,7 @@ schedule:
   - sles4sap/ensa/netweaver_ensa2_cluster
   - sles4sap/ensa/netweaver_ensa2_cluster_connector
```

Comment:
Not related to this PR but it might be good to rename ensa modules.
For example to differentiate between: 
`sles4sap/ensa/netweaver_ensa2_cluster_connector` - only setup
and
`sles4sap/sap_suse_cluster_connector` - test module

Reply:
Yes, I noticed this also. These two test modules are a little bit confusing. One is for setup one is for testing is a good idea.
I will open a new ticket for this.

Reply:
Opened [TEAM-9633](https://jira.suse.com/browse/TEAM-9633)
Rename ensa modules "netweaver_ensa2_cluster_connector" and "sap_suse_cluster_connector"